conjure-java-core/src/integrationInput/java/com/palantir/another/ServerEndpointSpecificErrors.java

```diff
@@ -0,0 +1,37 @@
+package com.palantir.another;
+
+import com.palantir.conjure.java.api.errors.CheckedServiceException;
+import javax.annotation.Nullable;
+import javax.annotation.processing.Generated;
+import org.jetbrains.annotations.Contract;
+
+@Generated("com.palantir.conjure.java.types.CheckedErrorGenerator")
+public final class ServerEndpointSpecificErrors {
```

Comment:
I think we need to delete the orphaned `Server`+namspace+`Errors` classes

Reply:
Yes we do, thanks. I wonder if there's a build time check we could add to ensure we don't accidentally check these in.